frontend/src/pages/EventSignInOptionsPage/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, useHistory } from 'react-router';
+import { Button } from '@material-ui/core';
+
+import useStyles from './styles';
+
+import * as ROUTES from '@Constants/routes';
+import { getEventById } from '@Services/EventService';
+import { EventResponse } from '@Services/api/models';
+
+interface EventID {
+  id: string;
+}
+
+function EventSignInOptionsPage(): JSX.Element {
+  const history = useHistory();
+  const { id } = useParams<EventID>();
+  const eventId = parseInt(id, 10);
+  const [eventInfo, setEventInfo] = useState<EventResponse | null>(null);
```

Comment:
Always put useState() calls at the top of the function body just so it looks nice